core/modules/layout/css/layout.admin.css

```diff
@@ -195,9 +214,11 @@
   float: left;
 }
 .layout-editor-block-content {
-  border: 1px solid #CCC;
+  background: #f7f7f7;
```

Comment:
I might be off on how Backdrop and Drupal define it in the CSS guidelines, I'm used to stricter rules on CSS style ordering that would list border margin and padding as box model styles.

If that's not how our CSS guidelines specify it, then ignore more :neckbeard: